src/pages/media.js

```diff
@@ -1,4 +1,20 @@
 import React from 'react';
 import Layout from '../components/dev-hub/layout';
+import useAllVideos from '../hooks/use-all-videos';
+import { P } from '../components/dev-hub/text';
+import CardList from '../components/dev-hub/card-list';
+import usePodcasts from '../hooks/use-podcasts';
 
-export default () => <Layout></Layout>;
+export default () => {
+    const { videos, error, isLoading } = useAllVideos();
+    const { podcasts, errorPodcasts, isLoadingPodcasts } = usePodcasts();
```

Comment:
```suggestion
    const { podcasts, error: errorPodcasts, isLoading: isLoadingPodcasts } = usePodcasts();
```

Here is one destructuring trick, so we get `error` and `isLoading` from podcasts but we have to reassign to a different variable (since we are using const) because otherwise these will be undefined.

Try logging them out to see we get `undefined` and not `null` or `true/false`!